examples/README.md

```diff
@@ -27,7 +27,7 @@ and stores (logs) them as MLflow artifacts.
 * `sklearn_logistic_regression` is a simple MLflow example with hooks to log training data to MLflow
 tracking server.
 * `supply_chain_security` shows how to strengthen the security of ML projects against supply-chain attacks by enforcing hash checks on Python packages.
-* `tensorflow` contains end-to-end one run examples from train to predict for both TensorFlow 1.X and 2.0. It includes usage of MLflow's
+* `tensorflow` contains end-to-end one run examples from train to predict for TensorFlow 2.6+ It includes usage of MLflow's
```

Comment:
Do we only support TensorFlow 2.6+?

Reply:
I think we support 2.0+? @WeichenXu123

Reply:
support 2.3+ 
keep consistent with mlflow master

Reply:
Can you fix it?

Reply:
fixed.